### ISSUE TITLE
Firefox Apparmor Settings to allow access to FreeTube

### ISSUE THREAD

**Issue by DPTJKKVH** [FreeTubeApp/FreeTube#923]
**Topic**
Please select what topic you are referring to.
- [ ] Coding
- [ ] Contribution
- [ ] Languages
- [X] Setup
- [ ] Other. Please specify:

**Question**
Hello everyone. I am on Ubuntu 20.10 and use the pre-installed Firefox, updated through the package manager. So no Snap install and confinement. I do however use the default shipped (but by default disabled) apparmor config for Firefox to isolate the browsers access to certain parts of my system. The default apparmor config ships with whitelists for typical stuff that a browser usually needs access to like the Downloads folder or optionally MPV and such things.

Long story short: I use the privacy redirect addon to redirect YouTube links to FreeTube but it doesn't work because it gets caught by apparmor. Does anybody know what exactly I need to whitelist in apparmor so I can successfully redirect to FreeTube? Bonus Karma for posting a working whitelist rule. Many thanks in advance! 

Also big thanks to everyone who has contributed to this wonderful piece of open source software. FreeTube is fantastic as it almost perfectly recreates the YouTube usage experience in a privacy friendly and secure way.

**Comment by DPTJKKVH** [comment]
Managed to "fix" it myself and happy to share here for everyone else who might wonder:

Modify `/etc/apparmor.d/usr.bin.firefox` with your favorite editor.

Add the following line:
`/opt/FreeTube/freetube ux,`

Enforce the new rule with `sudo aa-enforce /etc/apparmor.d/usr.bin.firefox`

Restart firefox if it was opened already.

**Comment by DPTJKKVH** [comment]
Hello @PrestonN . I wanted to add this information/guide to the Wiki so it can be easily found by others that use apparmor but it seems as if I can't create pull requests for the wiki. Can you please add the information above at your earliest convenience? Many thanks!
